<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalListWidget</class>
 <widget class="QWidget" name="TerminalListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>220</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>会场</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="mainToolBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="floatable">
      <bool>true</bool>
     </property>
     <addaction name="leaveConferenceAction"/>
     <addaction name="showAllTerminalsAction"/>
     <addaction name="showOnlineTerminalsAction"/>
     <addaction name="showOfflineTerminalsAction"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="conferenceTitleLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="memberVideoScrollArea">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>714</width>
           <height>208</height>
          </rect>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="showAllTerminalsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="CoolView.qrc">
     <normaloff>:/image/users_all.png</normaloff>:/image/users_all.png</iconset>
   </property>
   <property name="text">
    <string>显示全部终端</string>
   </property>
   <property name="toolTip">
    <string>显示全部终端</string>
   </property>
  </action>
  <action name="showOnlineTerminalsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="CoolView.qrc">
     <normaloff>:/image/online.png</normaloff>:/image/online.png</iconset>
   </property>
   <property name="text">
    <string>显示在线终端</string>
   </property>
   <property name="toolTip">
    <string>显示在线终端</string>
   </property>
  </action>
  <action name="showOfflineTerminalsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="CoolView.qrc">
     <normaloff>:/image/offline.png</normaloff>:/image/offline.png</iconset>
   </property>
   <property name="text">
    <string>显示离线终端</string>
   </property>
   <property name="toolTip">
    <string>显示离线终端</string>
   </property>
  </action>
  <action name="leaveConferenceAction">
   <property name="icon">
    <iconset resource="CoolView.qrc">
     <normaloff>:/image/left-arrow.png</normaloff>:/image/left-arrow.png</iconset>
   </property>
   <property name="text">
    <string>离开会议</string>
   </property>
   <property name="toolTip">
    <string>离开会议</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CoolView.qrc"/>
 </resources>
 <connections/>
</ui>
